### PR TITLE
Add guidelines to ensure the maintainers and issue jannies understand the place and purpose of templates.

### PR DESCRIPTION
**Pull request by MrsTonedOne**
## Why It's Good For The ~~Game~~ Repo

Preemptively preventing a fork threat.

### CODE REVIEW

**Comment by Kylerace** [comment]
🐝

**Comment by LemonInTheDark** [comment]
Deja vu

**Comment by san7890** [comment]
image

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -64,6 +64,7 @@ These are the few directives we have for project maintainers.
     - Try to get secondary maintainer approval before merging if you are able to.
   - PRs with empty commits intended to generate a changelog.
 - Do not merge PRs that contain content from the [banned content list](./CONTRIBUTING.md#banned-content).
+- Do not close PRs purely for breaking a template if the same information is contained without it.
```
Comment (LemonInTheDark) [inline .github/CONTRIBUTING.md:67]: A: got a lot of balls etc etc 
B: this is a kinda wordy way of getting this across. I also am not a huge fan of using this prescriptively

**Review by Kylerace — APPROVED** [review]

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -87,6 +88,7 @@ Things you **CAN** do:
 
 Things you **CAN'T** do:
 * [Close PRs](https://imgur.com/w2RqpX8.png): Only maintainers are allowed to close PRs. Do not hit that button.
+* Close issues purely for breaking a template if the same information is contained without it.
```
Comment (necromanceranne) [inline .github/CONTRIBUTING.md:91]: Okay but if something is conveyed in an extremely difficult to parse manner through the markdown text editor, yet still contains the relevant information, does this still allow for reformatting a pr/issue to make it more legible? What about attempts to revert a change to improve legibility back to the more illegible state, if a change is permitted?

**Comment by interestingusernam3** [comment]
Reproduction: roll the god rolls of a post skill issue based robusting and spawn on nukie shuttle

**Comment by optimumtact** [comment]
You could get rid of the entire file and just replace it with "assume good faith"

**Pull request merged**